packages/ui/feature-builder-form-controls/src/lib/pipes/should-show-dynamic-input-toggle-above-input.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { PieceProperty, PropertyType } from '@activepieces/pieces-framework';
+
+@Pipe({
+  name: 'shouldShwoDynamicInputToggleAboveInput',
+  pure: true,
+  standalone: true,
+})
+export class ShouldShwoDynamicInputToggleAboveInputPipe
+  implements PipeTransform
+{
+  transform(property: PieceProperty, isTrigger: boolean): boolean {
+    const allowedTypes = [
+      PropertyType.STATIC_DROPDOWN,
+      PropertyType.DROPDOWN,
+      PropertyType.STATIC_MULTI_SELECT_DROPDOWN,
+      PropertyType.MULTI_SELECT_DROPDOWN,
```

Comment:
What about dynamic values? Checkboxes? Array? Object? Json?

Reply:
They have the dynamic input icon passed to them through ng-content, because the dynamic input icon position in relation to the label differs.
Object, Array and JSON -> next to the label that's above the control.
Checkbox -> next to the toggle name.
Dynamic -> none, they show the dynamic input icons for each control inside of them.